### ISSUE TITLE
Meta: can we get some triage labels on community proposals?

### ISSUE THREAD

**Issue by orthoxerox** [dotnet/csharplang#459]
https://github.com/fsharp/fslang-suggestions has them, and it would be very useful to see at a glance what the position of the LDT is. Something along the lines of:

1. "Probably not" - it's unlikely the LDT will consider this
2. "Needs more info" - it's unclear what this proposal is about, more in-depth description is needed
3. "Needs more justification" - it's unclear what the benefits are
4. "LDM backlog" - the LDT will discuss this and will then assign labels 1-3 + "LDM verdict" or assign "Approved in principle" and champion it.

**Comment by gafter** [comment]
I don't see how we could get the position of the LDM (bullets 1 through 3) without the LDM discussing it. I also don't see us adding "LDM backlog" to issues unless it was already on current backlog or relevant to features currently in development.

In other words, we could do that, but I'm not sure it would help you very much, as very few issues would get these tags or are actually discussed in LDM meetings.

**Comment by alrz** [comment]
>very few issues would get these tags or are actually discussed in LDM meetings.

I think that's precisely what OP is asking for:

>to see at a glance what the position of the LDT is

So I think that'd be a good start.

**Comment by orthoxerox** [comment]
@gafter Yes, it probably helps that Don Syme is a BDFL. The tags could represent  the position of any LDT member who has actually read the proposal. I mean, *someone*  from the team has to actually like one of the proposals to bring it up during an LDM.

**Comment by HaloFour** [comment]
@gafter

Which brings up another problem.  There is a very wide gulf between the issues that are so interesting that they catch the attention of the LDM and the issues that are so obviously off-base that they can comfortably be immediately closed.  Everything in that middle seems to just fall into limbo and piles up.  It would be nice if there was some kind of process that could periodically review said issues and quickly triage if they're remotely actionable and/or what might need to happen before they might be considered further.

**Comment by alrz** [comment]
BTW, some possible "labels" are defined under ["Language team makes a decision"](https://github.com/dotnet/roslyn/blob/master/docs/contributing/Developing%20a%20Language%20Feature.md) in docs.

- Ready for prototype
- Not actionable
- Declined as proposed
- Backlog
- Won't Fix
